components/Navbar.tsx

```diff
@@ -69,14 +92,29 @@ const Navbar = ({ user, open, toggleDrawer }) => {
             Profile
           </Typography>
         </Grid>
-        <Grid xs={2} sx={{ paddingBottom: 1 }}>
+        <Grid
+          xs={2}
+          sx={{ paddingBottom: 1 }}
+          style={{
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'center',
+          }}
+        >
           <IconButton size="large">
             <LogoutRoundedIcon fontSize="large" sx={{ color: '#000' }} />
           </IconButton>
         </Grid>
       </Grid>
       <Drawer variant="temporary" open={open} onClose={toggleDrawer}>
-        <Box sx={{ width: '70vw', marginTop: '10vh', paddingLeft: 7 }}>
+        <Box
+          sx={{
+            width: isLarge ? '40vw' : '70vw',
```

Comment:
MUI support syntax like this:  `sx=[smallScreen, largeScreen]` you can find document in the `sx` property for Box and others.